src/QsCompiler/CompilationManager/EditorSupport/SyntaxUtils.cs

```diff
@@ -174,75 +173,35 @@ TypedExpression CondBlockExpression(Tuple<TypedExpression, QsPositionedBlock> b)
             };
         }
 
-        private static IEnumerable<LocalVariableDeclaration<string>> DeclarationsInExpressionByPosition(
+        private static IEnumerable<LocalVariableDeclaration<string, ResolvedType>> DeclarationsInExpressionByPosition(
             TypedExpression expression, Position position)
         {
             return expression.ExtractAll(Declarations);
 
-            IEnumerable<LocalVariableDeclaration<string>> Declarations(TypedExpression e)
+            IEnumerable<LocalVariableDeclaration<string, ResolvedType>> Declarations(TypedExpression e)
```

Comment:
nit: since this is only used here, I suggest using a lambda instead. GitHub won't let me add a code suggestion here, but something like
```
return expression.ExtractAll(e =>
{
    if (!e.Range.Any(r => r.Contains(position)))
    ...
});
```

Reply:
done